test/OrchardCore.Tests/OrchardCore.Tests.csproj

```diff
@@ -87,6 +85,10 @@
     </None>
   </ItemGroup>
 
+  <ItemGroup>
+    <Folder Include="OrchardCore.Users\Recipes\" />
+  </ItemGroup>
```

Comment:
Why is this needed?

Reply:
It's not.  I think because I removed the recipe file this was added because we now have empty folder. I'll remove it

Reply:
removed